container/dependencies.nix

```diff
@@ -85,7 +85,6 @@ let patched_rPackages =
         importlib-metadata
         ipykernel
         ipython
-        ipython_genutils
```

Comment:
why has this package been removed? I don't know what it's for so I'm not sure if it can be removed

Reply:
It's an "internals"-type package meant to be used only by ipython and similar, so it's pulled in as a dependency and hence isn't actually being removed, but users shouldn't depend on it.